src/Daire.java

```diff
@@ -0,0 +1,22 @@
+public class Daire {
+    double yaricap;
```

Comment:
yarıcap degerine nerden ulasmak istediğimizi belirtmek için ulasım belirteci belirlememişsiniz